src/modules/launcher/Wox.Test/FuzzyMatcherTest.cs

```diff
@@ -228,7 +230,7 @@ public void WhenMultipleResults_ExactMatchingResult_ShouldHaveGreatestScore(stri
 
         [TestCase("Windows Terminal", "Windows_Terminal", "term")]
         [TestCase("Windows Terminal", "WindowsTerminal", "term")]
-        public void FuzzyMatchingScore_ShouldBeHigher_WhenPreceedingCharacterIsSpace(string firstCompareStr, string secondCompareStr, string query)
+        public void FuzzyMatchingScoreShouldBeHigherWhenPreceedingCharacterIsSpace(string firstCompareStr, string secondCompareStr, string query)
```

Comment:
@ryanbodrug-microsoft FxCop here breaks our test naming convention. Do you think we should keep our convention and instead suppress this error for the entire project ?

Reply:
I think the naming is the same.  It's just removing the underscores.  I'm okay with this change as is, but I'm used to reading the "Should -> When" syntax.  My personal opinion would be to conform with broader conventions as those imposed by fxcop.